.github/workflows/Shared-Pre-Release-GH.yml

```diff
@@ -0,0 +1,70 @@
+# *************************************************************************
+# **  Copyright (c) 2022 CentraleSupélec & EDF.
+# **  All rights reserved. This program and the accompanying materials
+# **  are made available under the terms of the Eclipse Public License v2.0
+# **  which accompanies this distribution, and is available at
+# **  https://www.eclipse.org/legal/epl-v20.html
+# ** 
+# **  This file is part of the RiseClipse tool
+# **  
+# **  Contributors:
+# **      Computer Science Department, CentraleSupélec
+# **      EDF R&D
+# **  Contacts:
+# **      dominique.marcadet@centralesupelec.fr
+# **      aurelie.dehouck-neveu@edf.fr
+# **  Web site:
+# **      https://riseclipse.github.io
+# *************************************************************************
+
+name: Create pre-release on GitHub
+
+# This CD workflow creates a pre-release on GitHub
+# It aslo uploads artifacts if it's a tool
+
+on:
+  workflow_call:
+    inputs:
+      isTool:
```

Comment:
I am not sure to have a use-case for pre-releasing a component (yagni)

Reply:
If there is no need, I can delete this option. Workflow remains nearly unchanged

Reply:
yes, delete